src/org/opensolaris/opengrok/configuration/messages/ConfigMessage.java

```diff
@@ -182,7 +182,11 @@ protected void invokeSetter(Configuration config, String field, String value) th
                 | IllegalArgumentException ex) {
```

Comment:
added a comment

All exceptions in the invocation are propagated as the `InvocationTargetException` exception